### PR TITLE
add quizzes

### PR DESCRIPTION
**Pull request by shivamsoni00**

### CODE REVIEW

File: Components/quizzes/cate-0/cate0.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    width: 100%;
+    margin: 0 auto;
+}
+
+.mov{
+   
+    font-size: var(--cate-title-s);
+    font-weight: var(--cate-title-w);
+    color: var(--cate-txt-color);
+
```
Comment (anandkumarcoder) [inline Components/quizzes/cate-0/cate0.css:24]: Remove white spaces and use prettier

File: Components/quizzes/cate-0/cate0.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://amazing-kowalevski-11b361.netlify.app/root.css">
+    <link rel="stylesheet" href="./cate0.css" />
+    <title>Movies</title>
```
Comment (anandkumarcoder) [inline Components/quizzes/cate-0/cate0.html:9]: add author here

**Review by anandkumarcoder — COMMENTED** [review]
Remove white spaces and use prettier , changes those and merge

**Review by tewarimohit — APPROVED** [review]
Good to merge...

**Pull request merged**